bin/index.js

```diff
@@ -11,18 +16,24 @@ cloudinary.config({
 });
 
 const site_url = yargs.argv._[0];
+const current_date = Date.now();
+const template_name = yargs.argv._[1] || `${current_date}_template`;
 const stylesheet = "https://4ormat.github.io/template-screenshots/public/style.css";
 
 const desktop = "1440x2400";
 // Might have to consider 766px instead of 768px to fix tablet view
 const tablet = "766x100";
 const mobile = "390x800";
 
-const addUrl2PNGOptions = (url, device, options) => {
+const device_type = [
+  "desktop", "tablet", "mobile"
+]
+
+const addUrl2PNGOptions = (url, dimensions, options) => {
   const defaultOptions = {
-    viewport: device,
+    viewport: dimensions,
```

Comment:
technically, this changed in the last PR, but it's strange to have a dynamic value to the "default" options. 
Can the dimensions argument be part of the `options` (override options)?

this would change to
```suggestion
const addUrl2PNGOptions = (url, options) => {
  const defaultOptions = {
    viewport: device,
    viewport: "1440x2400", // or whatever default is appropriate
```

and the function calls below would look like

```javascript
addUrl2PNGOptions(site_url, { viewport: mobile })
```

what do you think?